plugins/base/src/test/resources/expect/signatureTest/out/images/logo-icon.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="26" height="26" viewBox="0 0 26 26" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Are you sure this is signature specific file? If it does not, it should definitely be moved out of this PR